### PR TITLE
Tina minneapolis

### PR DESCRIPTION
**Pull request by TinaJin-zz**

### CODE REVIEW

**Comment by hakkoymaz** [comment]
@TinaJin  🍑

File: Tina-Minneapolis.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Minneapolis
+## Hi
+
```
Comment (hakkoymaz) [inline Tina-Minneapolis.md:3]: Are there any to-go places near by?

Comment (TinaJin-zz) [reply Tina-Minneapolis.md:3]: haha moa

**Review by hakkoymaz — APPROVED** [review]
Ready for information desk!!

**Pull request merged**